packages/components/src/components/input-password/shadow.tsx

```diff
@@ -133,12 +134,27 @@ export class KolInputPassword implements InputPasswordAPI {
 							readOnly={this.state._readOnly}
 							required={this.state._required}
 							spellcheck="false"
-							type="password"
+							type={this._passwordVisible ? 'text' : 'password'}
 							value={this.state._value as string}
 							{...this.controller.onFacade}
 							onKeyDown={this.onKeyDown}
 							onInput={this.onInput}
 						/>
+						{this.ref?.value && this.ref.value.length > 0 ? (
```

Comment:
```suggestion
						{this.ref?.value?.length > 0 ? (
```
Kleine Vereinfachung